icon_validator/rules/workflow_validators/workflow_title_validator.py

```diff
@@ -0,0 +1,52 @@
+from icon_validator.rules.validator import KomandPluginValidator
+from icon_validator.exceptions import ValidationException
+
+
+class WorkflowTitleValidator(KomandPluginValidator):
+
+    def validate(self, spec):
+        """
+        Checks that title is not blank.
+        Checks that title dose not end with a period.
+        Checks that title dose not start with a lower case letter.
+        Checks that title dose not start with a space.
+        Checks that title is 6 words or less.
+        Checks that title is properly capitalized.
+        """
+        if "title" not in spec.spec_dictionary():
+            raise ValidationException("Plugin title is missing.")
+
+        title = spec.spec_dictionary()["title"]
+
+        if not isinstance(title, str):
+            raise ValidationException("Title must not be blank")
+        if title == "":
+            raise ValidationException("Title must not be blank")
+        if title.endswith("."):
+            raise ValidationException("Title ends with period when it should not.")
+        if title[0].islower():
+            # This plugin title is OK: minFraud
+            # This plugin title is OK: ifconfig.co
+            raise ValidationException("Title should not start with a lower case letter.")
+        if title[0].isspace():
+            raise ValidationException("Title should not start with a whitespace character.")
+        if len(title.split()) > 7:
+            raise ValidationException(f"Title is too long, 6 words or less: contains {str(len(title.split()))}")
```

Comment:
```suggestion
            raise ValidationException(f"Title is too long, 6 words or less: contains {title.count(" ")}")
```

Simpler and no need to wrap the count in a string.